test/integration/serverless/next.config.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
   },
   experimental: {
     publicDirectory: true
-  }
+  },
+  // make sure error isn't thrown from empty publicRuntimeConfig
+  publicRuntimeConfig: {}
```

Comment:
I think there's another test needed for not having a `publicRuntimeConfig` at all

Reply:
The [serverless api-support](https://github.com/zeit/next.js/blob/be3eab2ad7892ca9a5be38f133494e700875804a/test/integration/api-support/test/index.test.js#L278) tests should cover this case right?